packages/bits/src/lib/time-picker/time-picker.component.less

```diff
@@ -12,11 +12,21 @@
         position: relative;
     }
 
-    &__icon {
+    &__trigger {
         cursor: pointer;
         position: absolute;
         top: @nui-space-xs + @nui-space-xxs;
         right: @nui-space-xs + @nui-space-xxs;
+        outline: none;
+
+        &:focus-within .nui-timepicker__icon {
```

Comment:
you could have just a nested `:focus-within` inside the `&__icon` below.

I'm still concerned about the wrapper, lets think how we can go about it in a different way. 

Please check if the screen reader really reads the contents of the nested `svg`s. If not maybe the `aria-ignore` is not relevant for the icon host.

Reply:
thanks, reader doesn't go to svg, I removed wrapper.